seisflows/workflow/forward.py

```diff
@@ -339,7 +339,7 @@ def evaluate_initial_misfit(self):
 
         self.system.run(run_list, path_model=self.path.model_init,
                         save_residuals=os.path.join(self.path.eval_grad,
-                                                    "residuals.txt")
+                                                    "residuals_{src}_{it}_0.txt")
```

Comment:
I wonder if it would be more explicit to not pass `{it}` through as a string formatter here? Since it is hard-set as '1' in `evaluate_objective_function`, it might make it slightly more readable to set that here already. Similar in the Inversion class, we can already dictate the iteration and step count (see comments there). That way anyone in the future reading won't have to look at `evaluate_objective_function` to figure out what `it` should be.

e.g.,

```python 
        # `save_residuals` hard-coded for iteration 1 and step count 0
        # assuming workflows calling this function are evaluating the
        # very first round of misfit quantification (i.e., i01s00)
        self.system.run(run_list, path_model=self.path.model_init,
                        save_residuals=os.path.join(self.path.eval_grad,
                                                    "residuals_{src}_1_0.txt")
```